employee-rostering-backend/pom.xml

```diff
@@ -81,12 +81,6 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>junit</groupId>
-          <artifactId>junit</artifactId>
-        </exclusion>
-      </exclusions>
```

Comment:
This is not enough. You need to also remove all `*jupiter*` dependencies if you want to use Junit 4. The single test that's currently in the project doesn't run. I think it's because of the jupiter dependencies on classpath.

+1 for keeping Junit 4 during migration. Let's keep the Spring Boot migration scope limited. We'll upgrade to Junit 5 after that.

Reply:
When doing this, please also remove the `pitest-junit5-plugin` dependency from `pitest-maven` plugin in this `pom.xml`. It's only there to make Pitest run with Junit 5 but makes Pitest fail when using Junit 4.